ansible.cfg

```diff
@@ -1,4 +1,5 @@
 [defaults]
+inventory = hosts
```

Comment:
why is that required?

Reply:
It's not directly related to the non-root change.  It's just a default, so you don't have to specify `-i hosts` for each command.  It can be overridden though.

I can submit a one-liner PR for this separately, if needed.